From 64454729e88d3cb5888657e16666c2f62619f6e6 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@igalia.com>
Date: Thu, 13 Nov 2014 10:57:36 -0600
Subject: [PATCH] Disable DRI3

DRI3 is broken in Fedora, so we need this workaround.
---
 src/ephy-main.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/ephy-main.c b/src/ephy-main.c
index 12704ea..2dd214c 100644
--- a/src/ephy-main.c
+++ b/src/ephy-main.c
@@ -238,6 +238,9 @@ main (int argc,
   int status;
   EphyFileHelpersFlags flags;
 
+  /* Fedora enables DRI3 which breaks many web sites, e.g. DuckDuckGo and YouTube. */
+  g_setenv ("LIBGL_DRI3_DISABLE", "1", FALSE);
+
 #ifdef ENABLE_NLS
   /* Initialize the i18n stuff */
   bindtextdomain (GETTEXT_PACKAGE, GNOMELOCALEDIR);
-- 
1.9.3

